ca/config.go

```diff
@@ -155,7 +159,22 @@ func (s *SecurityConfig) UpdateRootCA(rootCA *RootCA, externalCARootPool *x509.C
 
 	s.rootCA = rootCA
 	s.externalCAClientRootPool = externalCARootPool
-	return s.updateTLSCredentials(s.certificate, s.issuerInfo)
+	err := s.updateTLSCredentials(s.certificate, s.issuerInfo)
+	if err == nil && s.queue != nil {
+		s.queue.Publish(&api.NodeTLSInfo{
+			TrustRoot:           s.rootCA.Certs,
+			CertIssuerPublicKey: s.issuerInfo.PublicKey,
+			CertIssuerSubject:   s.issuerInfo.Subject,
+		})
+	}
```

Comment:
Sure - I just updated it because just seemed a little weird to have the update be handled from a completely different function as well, but that's probably fine.